decomp2gef.py

```diff
@@ -602,7 +617,7 @@ def title(self):
         if self.ready_to_display:
             title = "decompiler:{:s}:{:s}:{:d}".format(self.decompiler.name, self.curr_func, self.curr_line+1)
         else:
-            title = "decomipler:{:s}:error".format(self.decompiler.name)
+            title = "decompuiler:{:s}:error".format(self.decompiler.name)
```

Comment:
```suggestion
            title = "decompiler:{:s}:error".format(self.decompiler.name)
```
Lol you placed a 'u' in decompiler